build.cmd

```diff
@@ -294,46 +294,44 @@ set FSHARP_TEST_SUITE_USE_NUNIT_RUNNER=true
 %_msbuildexe% fsharp\fsharp.tests.fsproj /p:Configuration=%BUILD_CONFIG%
 @if ERRORLEVEL 1 echo Error: fsharp cambridge tests for nunit failed && goto :failure
 
-call RunTests.cmd %BUILD_CONFIG_LOWER% fsharp %TEST_TAGS% 
-@if ERRORLEVEL 1 type testresults\fsharp_failures.log && echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWER% fsharp %TEST_TAGS%' failed && goto :failure
+call RunTests.cmd %BUILD_CONFIG_LOWERCASECASECASE% fsharp %TEST_TAGS% 
+@if ERRORLEVEL 1 type testresults\FSharpNunit_Error.log && echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWERCASECASE% fsharp %TEST_TAGS%' failed && goto :failure
 set FSHARP_TEST_SUITE_USE_NUNIT_RUNNER=
 )
 
 if '%TEST_FSHARPQA_SUITE%' == '1' (
-call RunTests.cmd %BUILD_CONFIG_LOWER% fsharpqa %TEST_TAGS% 
-@if ERRORLEVEL 1 type testresults\fsharpqa_failures.log && echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWER% fsharpqa %TEST_TAGS%' failed && goto :failure
+call RunTests.cmd %BUILD_CONFIG_LOWERCASECASE% fsharpqa %TEST_TAGS% 
+@if ERRORLEVEL 1 type testresults\fsharpqa_failures.log && echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWERCASECASE% fsharpqa %TEST_TAGS%' failed && goto :failure
 )
 
 if '%TEST_COMPILERUNIT%' == '1' (
-call RunTests.cmd %BUILD_CONFIG_LOWER% compilerunit %TEST_TAGS% 
-@if ERRORLEVEL 1 echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWER% compilerunit' failed && goto :failure
+call RunTests.cmd %BUILD_CONFIG_LOWERCASECASE% compilerunit %TEST_TAGS% 
+@if ERRORLEVEL 1 type testresults\CompilerUnit_net40_Error.log && echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWERCASE% compilerunit' failed && goto :failure
```

Comment:
maybe it's more safe if the goto it's not inline.

```batch
if ERRORLEVEL 1 (
    echo Error: 'RunTests.cmd %BUILD_CONFIG_LOWERCASE% compilerunit' failed
    type testresults\CompilerUnit_net40_Error.log
    goto :failure
)
```

like that is output file doesnt exists, the build fail

batch syntax is awful :cry: